build.sbt

```diff
@@ -7,17 +7,14 @@ lazy val root = (project in file(".")).
     inThisBuild(List(
       organization := "com.rhdzmota",
       scalaVersion := "2.12.6",
-      version      := "0.1.0-SNAPSHOT"
+      version      := "1.0.0"
```

Comment:
API versioning and project versions are different. You'd better not mixing them.

Reply:
That's correct. Following your comment on using **ordinal numbers** for the API versioning, I think that the project version can stay with the **semver specification**.